HomeWork_12/HW12.py

```diff
@@ -0,0 +1,73 @@
+import json
+from datetime import datetime
+def timer():
+    start = datetime()
+    def inner():
+        return datetime() - start
+    return inner
+
+def funcname(func):
+    def inner(*args,**kwargs):
+        print(f"Функція {func.__name__}")
+        return func(*args,**kwargs)
```

Comment:
you need to return inner from fucnname here (without calling:
```

def funcname(func):
    def inner(*args,**kwargs):
        print(f"Функція {func.__name__}")
        return func(*args,**kwargs)
    return inner
```

And you don't write to a file here, but you should according to the task